README.md

```diff
@@ -29,7 +41,7 @@ Clone repository to: $GOPATH/src/github.com/terraform-providers/terraform-provid
 
 ```
 $ mkdir -p $GOPATH/src/github.com/terraform-providers; cd $GOPATH/src/github.com/terraform-providers
-$ git clone git@github.com:terraform-providers/terraform-provider-bigip
+$ git clone git@github.com:F5Networks/terraform-provider-bigip
```

Comment:
why is the organization here changed to `F5Networks`? is this code no longer part of `terraform-providers`?

Reply:
fixed typo, code is part of terraform-providers.